### PR TITLE
removed username from siteGeoJSON

### PR DESCRIPTION
**Pull request by willmt80**
Changed the siteGeoJson query to not look for username. Not a huge performance increase but enough to be significant, particularly on the longest response

old:
current-latency

new:
new-latency

### CODE REVIEW

**Review by rymaju — APPROVED** [review]
LGTM

Make sure to run mvn spotless:apply for formatting

File: api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java [hunk 1]
```diff
@@ -45,22 +42,12 @@ public Boolean getTreePresent() {
     return treePresent;
   }
 
-  public Double getDiameter() {
+  public Double diameter() {
```
Comment (SofieDunt) [inline api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java:45]: What's the naming convention for this? (diameter vs getDiameter)

**Review by SofieDunt — APPROVED** [review]
👍

File: api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java (hunk 1 above)
Comment (willmt80) [reply api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java]: Removed this from feature

**Pull request merged**